src/main/kotlin/no/nav/familie/baks/mottak/søknad/kontantstøtte/domene/KontantstøtteSøknad.kt

```diff
@@ -0,0 +1,270 @@
+package no.nav.familie.baks.mottak.søknad.kontantstøtte.domene
+
+data class KontantstøtteSøknad(
```

Comment:
Trenger vi å ha disse to steder? Kunne det vært lagt inn i Familie-Kontrakter?

Reply:
Jo, absolutt, men vi landet på å ha det 2 steder frem til vi er mer sikre på at vi har landet kontrakten 👍